### PR TITLE
Merge auth_roles_test.py into auth_test.py

### PR DESCRIPTION
**Pull request by ptnapoleon**
@mambocab to review

### CODE REVIEW

File: auth_test.py [hunk 1]
```diff
@@ -660,6 +662,1230 @@ def data_resource_creator_permissions(creator, resource):
             permissions.append((creator, '<all functions in %s>' % keyspace, perm))
     return permissions
 
+# Second value is superuser status
+# Third value is login status, See #7653 for explanation.
+mike_role = ['mike', False, True, {}]
+role1_role = ['role1', False, False, {}]
+role2_role = ['role2', False, False, {}]
+cassandra_role = ['cassandra', True, True, {}]
```
Comment (mambocab) [inline auth_test.py]: aaaaahhhhhhhhhh

Comment (mambocab) [inline auth_test.py]: In particular, these should be `namedtuple`s

**Comment by mambocab** [comment]
In principle, this merge is fine for now, but at some point we should have a more detailed look and separate these out into submodules in a single folder.

**Comment by mambocab** [comment]
Execution looks good too. I'm +1 on the changes; please either make those lists `namedtuple`s or file a JIRA for us to do that in this sprint.

**Comment by ptnapoleon** [comment]
Filed and assigned to myself.

**Pull request merged**